test/docker-compose.yml

```diff
@@ -173,6 +173,8 @@ services:
   zkevm-aggregator:
     container_name: zkevm-aggregator
     image: zkevm-node
+    extra_hosts:
```

Comment:
These docker compose configs might not be correct. Since i need to figure out which service needs host gateway.